src/main/java/ch/njol/skript/doc/JSONGenerator.java

```diff
@@ -188,14 +233,11 @@ private JsonObject generateFunctionElement(JavaFunction<?> function) {
 		functionJsonObject.addProperty("id", DocumentationIdProvider.getId(function));
 		functionJsonObject.addProperty("name", function.getName());
 		functionJsonObject.addProperty("since", function.getSince());
+		functionJsonObject.addProperty("return-type", getClassInfoName(function.getReturnType()));
```

Comment:
I think it could be great if we did the same thing with providing the ID here too